packages/turndown/src/turndown.js

```diff
@@ -216,15 +215,10 @@ function replacementForNode (node) {
   var whitespace = node.flankingWhitespace
   if (whitespace.leading || whitespace.trailing) content = content.trim()
 
-  const replaceNonbreakingSpaces = space => {
-    // \u{00A0} is a nonbreaking space
-    return space.replace(/\u{00A0}/ug, this.options.nonbreakingSpace);
-  };
-
   return (
-    replaceNonbreakingSpaces(whitespace.leading) +
-    replaceNonbreakingSpaces(rule.replacement(content, node, this.options)) +
-    replaceNonbreakingSpaces(whitespace.trailing)
```

Comment:
`replaceNonbreakingSpaces` was not present in upstream turndown and was added by #8468. It is no longer used and has thus been removed.